packages/backend/src/utils/validate.ts

```diff
@@ -0,0 +1,11 @@
+import { Types } from 'mongoose'
+
+export const isValidId = (id: string) =>
+  Types.ObjectId.isValid(id) ? true : false
+
+export const isUndefined = (item: any) =>
+  typeof item === 'undefined' ? true : false
+export const isString = (item: any) => (typeof item === 'string' ? true : false)
+
+export const isBoolean = (item: any) =>
+  typeof item === 'boolean' ? true : false
```

Comment:
Are all `x ? true : false` not redundant? If the case (in JS too), then these can all be simplified to simply `x`

Example:
```
export const isUndefined = (item: any) => 
  typeof item === 'undefined'
```